spec/carin_for_blue_button/custom_groups/eob/insurer_same_test_spec.rb

```diff
@@ -0,0 +1,82 @@
+RSpec.describe CarinForBlueButtonTestKit::CARIN4BBV200::InsurerSameTest do
+  let(:eob_json_string) do
+    File.read(File.join(__dir__, '..', '..', '..', 'fixtures', 'c4bb_eob_inpatient_example.json'))
+  end
+  let(:organization_json_string) do
+    File.read(File.join(__dir__, '..', '..', '..', 'fixtures', 'c4bb_organization_example.json'))
+  end
+  let(:coverage_json_string) do
+    File.read(File.join(__dir__, '..', '..', '..', 'fixtures', 'c4bb_coverage_example.json'))
+  end
+  let(:coverage_secondary_json_string) do
+    File.read(File.join(__dir__, '..', '..', '..', 'fixtures', 'c4bb_eob_coverage_example.json'))
+  end
+
+  let(:suite) { Inferno::Repositories::TestSuites.new.find('c4bb_v200') }
+  let(:session_data_repo) { Inferno::Repositories::SessionData.new }
+  let(:test_session) { repo_create(:test_session, test_suite_id: suite.id) }
+  let(:url) { 'http://example.com/fhir' }
+  let(:error_outcome) { FHIR::OperationOutcome.new(issue: [{ severity: 'error' }]) }
+  def run(runnable, inputs = {})
```

Comment:
If this PR is still going to be merged, then this method should be removed since its defined in Inferno Core. See #79 for an example